holodex/dhen-data/org.json

```diff
@@ -153,29 +144,23 @@
                     "name": "trainee",
                     "pluralName": "trainees",
                     "relationshipType": "relationshipTypes/Trainer",
-                    "linkType": "linkTypes/trainee-of",
-                    "label": "role 2 label"
+                    "labels": " a trainee of {relation}"
                 }
             ],
             "links": [
                 {
                     "@type": "LinkType",
-                    "id": "linkTypes/trainer-for",
+                    "id": "linkTypes/trains",
                     "relationshipType": "relationshipTypes/Trainer",
                     "source": "roleTypes/trainer",
                     "target": "roleTypes/trainee",
-                    "label": "link 1 label"
-                },
-                {
-                    "@type": "LinkType",
-                    "id": "linkTypes/trainee-of",
-                    "relationshipType": "relationshipTypes/Trainer",
-                    "source": "roleTypes/trainee",
-                    "target": "roleTypes/trainer",
-                    "label": "link 2 label"
+                    "labels": "{source} trains {target}",
+                    "queryLabels": "People who train {target}, People trained by {source}, People whom {source} trains, People who train others for {context}"
                 }
             ]
         },
+        /* Question: Sites may be the context for relationships?*/
+
```

Comment:
@simontegg @ahdinosaur Sorry for the delay, was fixing data in my local database in prep for re-generating, but my linux computer seems to be dead, I am temporarily out of commission for coding.  

In this case, a Harvesting Site (actually a farm usually, although it could be a park or something like that) has the relationship with the herbal network.  Just says X is a harvesting site for the network, so the context is one side of the relationship.  And the context would be the herbal network (DHEN), not the harvesting site.

The data here is not right, the harvester is not actually involved in this relationship, sorry for the confusion.

We do have some other instances where we need the context in addition to the 2 ends of the relationship.  Although we haven't addressed this in our software yet, since there is no confusion at the moment.

Even though this is bad data, I'm wondering if there an implication we should discuss?  Like is there something we should do in the model when the context is one side of the relationship; and when it is not?